lib/searchable_select.ex

```diff
@@ -74,9 +77,12 @@ defmodule SearchableSelect do
     |> assign(:search, "")
     |> assign(:selected, assigns[:selected] || [])
     |> assign(:value_callback, assigns[:value_callback] || fn item -> item.id end)
+    |> assign(:sort_mapper, assigns[:sort_mapper])
```

Comment:
Could this be called `:sort_callback`/`:sorting_callback` to match the other callback assigns here (`:label_callback` and `:value_callback`)
🙏

Reply:
done